docs/how-to-use-relational-backend-storage.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to use relational backend storage
+slug: /how-to-use-relational-backend-storage
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
+Before the version `0.5.0`, Gravitino only supports KV backend storage to store metadata. Since
+RDBMS is widely used in the industry, starting from the version `0.5.0`, Gravitino supports using
+RDBMS as relational backend storage to store metadata. This doc will guide you on how to use the
+relational backend storage in Gravitino.
+
+Relational backend storage mainly aims to the users who are accustomed to using `RDBMS` to
```

Comment:
Should we keep RDBMS all-emphasis or none-emphasis?

Reply:
I think it is not necessary.